dynawo/sources/Modeler/DataInterface/DYNGeneratorInterfaceIIDM.cpp

```diff
@@ -113,6 +115,10 @@ GeneratorInterfaceIIDM::importStaticParameters() {
     staticParameters_.insert(std::make_pair("uc", StaticParameter("uc", StaticParameter::DOUBLE).setValue(U0)));
     staticParameters_.insert(std::make_pair("angle", StaticParameter("angle", StaticParameter::DOUBLE).setValue(teta)));
     staticParameters_.insert(std::make_pair("vNom", StaticParameter("vNom", StaticParameter::DOUBLE).setValue(vNom)));
+    staticParameters_.insert(std::make_pair("targetV_pu", StaticParameter("targetV_pu", StaticParameter::DOUBLE).setValue(getTargetV()/vNom)));
```

Comment:
Maybe getTargetV() / vNom instead of getTargetV()/vNom to be consistent with the other equations

Reply:
ok, thanks